src/app/portal/services/mapping/mapping.service.spec.ts

```diff
@@ -13,4 +16,74 @@ describe('MappingService', () => {
   it('should be created', () => {
     expect(service).toBeTruthy();
   });
+
+  describe('#mapTranscriptRequestToRows', () => {
+    it('should map correctly', () => {
+      const hearingDateDateTime = DateTime.fromISO('2024-02-01');
+      const requestedOnDateTime = DateTime.fromISO('2024-01-01');
```

Comment:
I think it may be better if these variables were removed and put in-line with the properties to make it more readable

Reply:
Fixed